SPECS/xmlstarlet/xmlstarlet.spec

```diff
@@ -0,0 +1,208 @@
+%{!?_pkgdocdir: %global _pkgdocdir %{_docdir}/%{name}-%{version}}
+
+Summary:        Command Line XML Toolkit
+Name:           xmlstarlet
+Version:        1.6.1
+Release:        16%{?dist}
+License:        MIT
+URL:            http://xmlstar.sourceforge.net/
+Vendor:         Microsoft
+Distribution:   Mariner
+Source0:        https://downloads.sourceforge.net/project/xmlstar/%{name}/%{version}/%{name}-%{version}.tar.gz
+
+# https://sourceforge.net/p/xmlstar/bugs/109/
+Patch0:         xmlstarlet-1.6.1-nogit.patch
+# http://sourceforge.net/tracker/?func=detail&aid=3266898&group_id=66612&atid=515106
+
+BuildRequires:  gcc
+BuildRequires:  xmlto
+BuildRequires:  automake 
+BuildRequires:  autoconf
+BuildRequires:  libxslt-devel
+BuildRequires:  libxml2-devel >= 2.6.23
+BuildRequires:  docbook5-schemas
+
+
+%description
+XMLStarlet is a set of command line utilities which can be used
+to transform, query, validate, and edit XML documents and files
+using simple set of shell commands in similar way it is done for
+plain text files using UNIX grep, sed, awk, diff, patch, join, etc
+commands.
+
+%prep
+%setup -q
+%patch0 -p1 -b .nogit
+
+
+%build
+autoreconf -i
+%configure --disable-static-libs --with-libxml-include-prefix=%{_includedir}/libxml2 --docdir=%{_pkgdocdir} # --libdir=%{_libdir}
+make %{?_smp_mflags}
+
+
+%install
+rm -fr %{buildroot}
+make install DESTDIR=$RPM_BUILD_ROOT
+# Avoid name kludging in autotools
+mv $RPM_BUILD_ROOT%{_bindir}/xml $RPM_BUILD_ROOT%{_bindir}/xmlstarlet
```

Comment:
>$RPM_BUILD_ROOT [](start = 3, length = 15)

Please change instances of `$RPM_BUILD_ROOT` to `${buildroot}`, since they're equivalent and we've standardized on the shorter version.

Reply:
Done, changed it in all the files currently in the PR